Vagrantfile

```diff
@@ -102,7 +102,7 @@ Vagrant.configure("2") do |config|
     if ENV['OSQUERY_BUILD_CPUS']
       v.cpus = ENV['OSQUERY_BUILD_CPUS'].to_i
     else
-      v.cpus = 2
+      v.cpus = 4
```

Comment:
What's wrong with defining `OSQUERY_BUILD_CPUS=4 vagrant up`? `reload` with that defined will also reconfigure the host to have the new set of cores.